src/sync/sync-utils.ts

```diff
@@ -54,13 +55,22 @@ export const findOrStartSync = async (
 	}, 0, logger);
 };
 
+export const convertCommitsFromDateStringToDate = (commitsFromDate: string | undefined, logger: Logger) => {
```

Comment:
This function make sense here because originally the string is converted from a date above `commitsFromDate: commitsFromDate?.toISOString(),`

Reply:
They should be co-locate.